bridge/client/app/_components/ktb-loading-spinner/ktb-loading-spinner.component.scss

```diff
@@ -0,0 +1,84 @@
+@import '~@dynatrace/barista-components/core/src/style/variables';
+
+$ktb-loading-spinner-default-size: 30px;
+$ktb-loading-spinner-small-size: 15px;
+$ktb-loading-spinner-offset: 22px * 3.1416; // r(-gap)*2*3.14
+$ktb-loading-spinner-duration: 1700ms;
+$ktb-loading-spinner-ease: cubic-bezier(0.645, 0.045, 0.355, 1);
+
+:host {
+  display: inline-block;
+  width: $ktb-loading-spinner-default-size;
+  height: $ktb-loading-spinner-default-size;
+}
+
+// There is an issue in angular with the combination of :host ans :host-context.
+// As a workaround you have to move the :host-context after the :host selector
+:host :host-context(.dt-form-field),
+:host :host-context(.dt-filter-field),
+:host :host-context(.dt-button) {
+  width: $ktb-loading-spinner-small-size;
+  height: $ktb-loading-spinner-small-size;
+
+  // This overrides the min-with requirement for prefix/suffix
+  min-width: $ktb-loading-spinner-small-size;
+}
+
+.ktb-loading-spinner-svg {
+  display: block;
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  stroke-width: 4px;
+  stroke: $logolime;
+  fill: none;
+  animation: ktb-loading-spinner-spin $ktb-loading-spinner-duration * 2 linear infinite;
+}
+
+:host-context(.dt-form-field) .ktb-loading-spinner-svg[dtPrefix],
+:host-context(.dt-form-field) .ktb-loading-spinner-svg[dtSuffix],
+:host-context(.dt-filter-field) .ktb-loading-spinner-svg {
+  stroke: $gray-500;
+}
+
+:host-context(.dt-button) {
+  // align icon with the baseline of the text
+  vertical-align: -3px;
+  margin-right: 8px;
+
+  .ktb-loading-spinner-svg {
+    stroke: #ffffff;
+  }
+}
+
+:host-context(.dt-button-secondary) .ktb-loading-spinner-svg {
+  stroke: $gray-300;
```

Comment:
Also here please use a css variable, e.g. `var(--my-loading-spinner-button-secondary-color)` so that this can be adjusted via custom theme, because the color of the Button might also be different, depending on the theming.

Reply:
As discussed this color will stay hard-coded.